docs/developer-docs/smart-contracts/advanced-features/serving-http-request.mdx

```diff
@@ -111,7 +111,17 @@ fn http_request(req: HttpRequest) -> HttpResponse {
 
 </TabItem>
 
-<TabItem value="typescript" label="Typescript">
+<AdornedTab value={"typescript"} label="TypeScript" endAdornment={<BetaChip />}>
+
+To serve HTTP requests, you can use the Azle `http-server` functionality by creating a new project with the flag `--http-server`:
```

Comment:
Like we discussed in Slack, perhaps for now you want to use the `http_request` and `http_request_update` methods, similar to the previous `Canister` example. And the dev probably shouldn't be introduced to `--http-server` right now, unless we're really ready, from DFINITY's side, to introduce the concept.

We are introducing the concept in The Azle Book as a core distinction between canister types, those being Candid RPC canisters (everything DFINITY focuses on and the focus of the 1.0 syntax), and HTTP Server canisters (which don't use IC-specific APIs as far as possible and we believe should be a long-term goal).

Reply:
revised example